src/utils.ts

```diff
@@ -58,7 +69,7 @@ if (raf && !caf) {
 } else if (!(raf && caf)) {
 	raf = function(callback) {
 		return window.setTimeout(function() {
-			callback(window.performance && window.performance.now());
+			callback(window.performance && window.performance.now && window.performance.now());
```

Comment:
`callback` function expects timestamp as a parameter. But if `window.performance.now` is undefined, undefined is passed as a parameter. Is it OK to pass `undefine`?